src/llama-kv-cache-recurrent.cpp

```diff
@@ -88,8 +94,8 @@ llama_kv_cache_recurrent::llama_kv_cache_recurrent(
         ggml_tensor * v = ggml_new_tensor_1d(ctx, type_v, n_embd_v_gqa*kv_size);
         ggml_format_name(k, "cache_k_l%d", i);
         ggml_format_name(v, "cache_v_l%d", i);
-        k_l.push_back(k);
```

Comment:
This `push_back` was broken with the addition of the filter above to skip non-recurrent layers.